config/rector/sets/cakephp60.php

```diff
@@ -532,4 +509,90 @@
             }
         }
     }
+
+    // Rename methods that start with an underscore to a new, non-conflicting name.
+
+    $map = [
+        'Console' => [
+            'Cake\Console\ConsoleOutput' => [
+                '_write' => 'writeStream',
+            ],
+        ],
+        'Form' => [
+            'Cake\Form\Form' => [
+                '_execute' => 'process',
+            ],
+        ],
+        'Http' => [
+            'Cake\Http\Client' => [
+                '_sendRequest' => 'processRequest',
+            ],
+            'Cake\Http\ServerRequest' => [
+                '_is' => 'isType',
+            ],
+            'Cake\Http\Client\Adapter\Stream' => [
+                '_send' => 'processRequest',
+            ],
+        ],
+        'I18n' => [
+            'Cake\I18n\DateFormatTrait' => [
+                '_parseDateTime' => 'processDateTime',
+            ],
+        ],
+        'Mailer' => [
+            'Cake\Mailer\Transport\SmtpTransport' => [
+                '_connect' => 'connectSmtp',
+                '_disconnect' => 'disconnectSmtp',
+            ],
+        ],
+        'ORM' => [
+            'Cake\ORM\Table' => [
+                '_saveMany' => 'doSaveMany',
+                '_deleteMany' => 'doDeleteMany',
+            ],
+            'Cake\ORM\Behavior\TreeBehavior' => [
+                '_moveUp' => 'doMoveUp',
+                '_moveDown' => 'doMoveDown',
+                '_removeFromTree' => 'doRemoveFromTree',
+            ],
+            'Cake\ORM\Association\HasMany' => [
+                '_unlink' => 'doUnlink',
+            ],
+            'Cake\ORM\Query\SelectQuery' => [
+                '_decorateResults' => 'ormDecorateResults',
+                '_execute' => 'ormExecute',
```

Comment:
i added the `orm` prefix since both those methods exist in the `\Cake\Database\Query\SelectQuery` as well.